src/x86_64/périphériques/RTC.md

```diff
@@ -13,19 +13,27 @@ Elle peut être utilisée pour avoir la date et heure précise. (voir ACPI pour
 # Lire le temps
 Il est possible de lire depuis la RTC en utilisant les fonctions suivantes:
 ```c
+
+enum 
+{
```

Comment:
in the documentation, put a markdown table instead of a C code enum, it's better for reading

Reply:
and add other register, like STATUS_REGISTER_B, week, day, hour, ... in the table  (maybe ?)